pilot/pkg/config/kube/ingress/controller.go

```diff
@@ -361,6 +373,9 @@ func (c *controller) List(typ config.GroupVersionKind, namespace string) ([]conf
 	ingressByHost := map[string]*config.Config{}
 
 	for _, ingress := range sortIngressByCreationTime(c.ingressInformer.GetStore().List()) {
+		if c.namespacesFilter != nil && !c.namespacesFilter(ingress) {
```

Comment:
I think if we use NewFilteredSharedIndexInformer List will handle this automatically?

Reply:
good catch, i will move it to a common place